x/programs/rust/README.md

```diff
@@ -34,9 +128,30 @@ This folder provides the necessary tools to build WASM programs using rust.
 - `/host` : Imports necessary functions from the host.
 - `/Program`: Defines the `ProgramValue` and `Progam` types.
 
-### Expose Macro
-
-A rust crate that contains an attribute procedural macro `expose` allowing program functions to be exposed to the host.
+### Exporting Program Functions
+
+A program exposes functions that can be called by the host called exports. To
```

Comment:
```suggestion
A program exposes functions that can be called by the host. To
```
? "Called" twice like that is kind of confusing. Maybe `named` or something like that?